tests/c_api_tests/test_c_api_model.cpp

```diff
@@ -218,7 +218,7 @@ TEST_CASE("C API Model") {
         }
 
         SUBCASE("Tap changing strategy") {
-            PGM_set_tap_changing_strategy(hl, opt, PGM_tap_changing_strategy_any_valid_tap);
+            PGM_set_tap_changing_strategy(hl, opt, PGM_tap_changing_strategy_min_voltage_tap);
```

Comment:
my guess is that this is a safety measure to make sure the C API test has the same behavior? if so, that's alright; just want to understand this change.

Reply:
Since this is `check_no_throw`, it makes no sense to still check what we know to fail (`any`)

Reply:
why does it fail? shouldn't it have kept working the way it did before?

Reply:
Optimizer will throw error if the combination of `TapOptimizationStrategy::any` and `SearchMethod::binary_search` is used.